src/model/window-guardian.ts

```diff
@@ -118,6 +118,9 @@ export const makeGuardianBrowserCAPI = (CAPI: CAPIType): CAPIBrowserType => {
             discussionApiClientHeader: CAPI.config.discussionApiClientHeader,
 
             dcrSentryDsn: CAPI.config.dcrSentryDsn,
+
+            // ab tests
+            switches: CAPI.config.switches,
```

Comment:
I think this is the right call, `// switches` should have never been flattered into config.